src/main/java/com/mg105/entities/GiveTutorial.java

```diff
@@ -23,11 +23,11 @@ public GiveTutorial(boolean moved, boolean attacked, boolean usedItem) {
      * @param action the action that has been performed
      */
     public void ActionPerformedSetter(String action) {
-        if (action.equalsIgnoreCase(TutorialTexts.moved)) {
+        if (action.equalsIgnoreCase(TutorialTexts.MOVED)) {
```

Comment:
We usually write methods in `camelCase` not `PascalCase`

Reply:
does it even count as a method if its just a constant? btw I dont know if I was supposed to directly get the strings from Utils.TutorialTexts like this.

Reply:
Oh this comment is on the wrong line, I'm talking about `ActionPerformedSetter`.